ui/app/AppLayouts/Chat/controls/community/PermissionTypes.qml

```diff
@@ -6,7 +6,7 @@ import StatusQ.Core.Theme 0.1
 
 QtObject {
     enum Type {
-        None, Admin, Member, Moderator, ViewAndPost, Read
+        None, Admin, Member, Read, ViewAndPost
```

Comment:
Duplicate of what you defined in Constants below? 

(imo this should just move to constants)

Reply:
Hm.. not sure I understand.. so these things existed before, I just ensured the naming is semantically correct and we drop the moderation altogether